src/TokenHandler/AccountManager.mo

```diff
@@ -183,6 +188,14 @@ module {
           totalConsolidated_ += transferAmount;
           log(p, #consolidated({ deducted = deposit; credited = transferAmount }));
         };
+        case (#Err(#BadFee({ expected_fee }))) {
+          let err = #ExtendedBadFee({
+            original_fee = originalFee;
+            expected_fee = expected_fee;
+          });
+          log(p, #consolidationError(err));
+          return #Err(err);
```

Comment:
We don't need to log the #ExtendedBadFee anymore, so we can remove #ExtendedBadFee entirely. Logging the consolidation failure is enough. We are already logging old and new fee inside `setNewFee` with #feeUpdated.